src/khepri.erl

```diff
@@ -2594,8 +2597,162 @@ wait_for_async_ret(Correlation, Timeout) ->
+
+export(Module, ModulePriv) when is_atom(Module) ->
+    StoreId = khepri_cluster:get_default_store_id(),
+    export(StoreId, Module, ModulePriv).
+
+-spec export(StoreId | PathPattern, Module, ModulePriv) -> Ret when
+      StoreId :: khepri:store_id(),
+      PathPattern :: khepri_path:pattern(),
+      Module :: module(),
+      ModulePriv :: khepri_import_export:module_priv(),
+      Ret :: ok | {ok, ModulePriv} | {error, any()}.
+%% @doc Exports a Khepri store using the `Module' callback module.
+%%
+%% This function accepts the following two forms:
+%% <ul>
+%% <li>`export(StoreId, Module, ModulePriv)'. Calling it is the same as
+%% calling `export(StoreId, "**", Module, ModulePriv)'.</li>
+%% <li>`export(PathPattern, Module, ModulePriv)'. Calling it is the same as
+%% calling `export(StoreId, PathPattern, Module, ModulePriv)' with the default
+%% store ID (see {@link khepri_cluster:get_default_store_id/0}).</li>
+%% </ul>
+%%
+%% @see export/4.
+
+export(StoreId, Module, ModulePriv)
+  when?IS_STORE_ID(StoreId) andalso is_atom(Module) ->
+    export(StoreId, [?KHEPRI_WILDCARD_STAR_STAR], Module, ModulePriv);
+export(PathPattern, Module, ModulePriv)
+  when is_atom(Module) ->
+    StoreId = khepri_cluster:get_default_store_id(),
+    export(StoreId, PathPattern, Module, ModulePriv).
+
+-spec export(StoreId, PathPattern, Module, ModulePriv) -> Ret when
+      StoreId :: khepri:store_id(),
+      PathPattern :: khepri_path:pattern(),
+      Module :: module(),
+      ModulePriv :: khepri_import_export:module_priv(),
+      Ret :: ok | {ok, ModulePriv} | {error, any()}.
+%% @doc Exports a Khepri store using the `Module' callback module.
+%%
+%% The `PathPattern' allows to filter which tree nodes are exported. The path
+%% pattern can be provided as a native path pattern (a list of tree node names
+%% and conditions) or as a string. See {@link khepri_path:from_string/1}.
+%%
+%% `Module' is the callback module called to perform the actual export. It
+%% must conform to the Mnesia Backup &amp; Restore API. See {@link
+%% khepri_import_export} for more details.
+%%
+%% `ModulePriv' is the term passed to `Module:open_write/1'.
+%%
+%% Example: export the full Khepri store using {@link khepri_export_erlang} as
+%% the callback module
+%% ```
+%% ok = khepri:export(StoreId, khepri_export_erlang, "export-1.erl").
+%% '''
+%%
+%% Example: export a subset of the Khepri store
+%% ```
+%% ok = khepri:export(
+%%        StoreId,
+%%        "/:stock/:wood/**",
+%%        khepri_export_erlang,
+%%        "export-wodd-stock-1.erl").
+%% '''
+%%
+%% @param StoreId the name of the Khepri store.
+%% @param PathPattern the path pattern matching the tree nodes to export.
+%% @param Module the callback module to use to export.
+%% @param ModulePriv arguments passed to `Module:open_write/1'.
+%%
+%% @returns `ok' or an `{ok, Term}' tuple if the export succeeded (the actual
+%% return value depends on wheter the callback module wants to return anything
```

Comment:
```suggestion
%% return value depends on whether the callback module wants to return anything
```

looks like there's one more of these down on L2427

Reply:
Fixed.